src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -32,37 +51,48 @@ public class Vision {
 	//relative position of the camera on the robot ot the robot center
 	private final Transform3d m_robotToCam = new Transform3d(new Translation3d(Constants.CAMERA_X_OFFSET, Constants.CAMERA_Y_OFFSET, Constants.CAMERA_Z_OFFSET), new Rotation3d(Constants.CAMERA_ROLL_OFFSET,Constants.CAMERA_PITCH_OFFSET,Constants.CAMERA_YAW_OFFSET)); //Cam mounted facing forward, half a meter forward of center, half a meter up from center.
 
-	PhotonPoseEstimator m_photonPoseEstimator;
+	private PhotonPoseEstimator m_photonPoseEstimator;
+
+	private double m_lastImageTimeStamp = -1.0;
 
 	public Vision() {
-		try{
-			m_aprilTagFieldLayout = AprilTagFieldLayout.loadFromResource(AprilTagFields.k2023ChargedUp.m_resourceFile);
-		} catch(IOException e){
-			System.out.println("Unable to load AprilTag layout" + e.getMessage());
-			m_aprilTagFieldLayout = null;
-		}
+		// try{
+		// 	m_aprilTagFieldLayout = AprilTagFieldLayout.loadFromResource(AprilTagFields.k2023ChargedUp.m_resourceFile);
+		// } catch(IOException e){
+		// 	System.out.println("Unable to load AprilTag layout" + e.getMessage());
+		// 	m_aprilTagFieldLayout = null;
+		// }
+		m_aprilTagFieldLayout = SHED_TAG_FIELD_LAYOUT;
```

Comment:
Jack, maybe it would be better to use an if/else, with just true or false in the if, instead of commenting it in/out. Also, we might want one print statement to indicate which field layout is being used. That might help us when we are confused?

Reply:
I did notice that VSCode/Java complain if code is completely unreachable. This is not the same as other languages where a  "if (false) {.....}" block is understood to be effectively a comment, and it does not complain. So, maybe just using comments is cleaner. But a println still might be helpful, and it would only be 1 line.